owtf/webapp/src/containers/SettingsPage/index.js

```diff
@@ -2,18 +2,17 @@
  * SettingsPage
  */
 import React from 'react';
-import UnderconstructionPage from "components/UnderconstructionPage";
 import { Button , Alert } from 'react-bootstrap';
-import {Grid, Panel, Col, Row, FormGroup, Form, ControlLabel, Nav, NavItem} from 'react-bootstrap';
+import {Grid, Col, Row, FormGroup, Form, ControlLabel, Nav, NavItem} from 'react-bootstrap';
 import { Tabs, Tab , TabContainer, TabContent, TabPane } from 'react-bootstrap';
 
 import PropTypes from 'prop-types';
 import { connect } from 'react-redux';
 import { createStructuredSelector } from 'reselect';
 import { makeSelectFetchError, makeSelectFetchLoading, makeSelectFetchConfigurations, makeSelectChangeError } from './selectors';
 import { loadConfigurations, changeConfigurations } from "./actions";
-import InputGroup from "react-bootstrap/es/InputGroup";
 import FormControl from "react-bootstrap/es/FormControl";
+import { ClipLoader } from 'react-spinners';
```

Comment:
Try to use a CSS based one rather than depending on an external package - adds unnecessary bloat to the application.